tests/test_message.py

```diff
@@ -70,11 +70,7 @@
 from telegram.ext import Defaults
 from telegram.warnings import PTBDeprecationWarning
 from tests._passport.test_passport import RAW_PASSPORT_DATA
-from tests.auxil.bot_method_checks import (
-    check_defaults_handling,
-    check_shortcut_call,
-    check_shortcut_signature,
-)
```

Comment:
I left `check_defaults_handling` in here on purpose - I just commented out the usages so that we can see the code coverage reports for once ;) Please re-add and un-comment the usages